.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "pybind11"]
 	path = external/pybind11
 	url = https://github.com/pybind/pybind11.git
+	branch = v2.10.1
```

Comment:
Yes, this is for preventing possible errors in the future. It is usually just a good practice, but with Python it is crucial because the support of lower Python versions can be dropped in further releases